### DISCUSSION TITLE
TFT35 E3 V3.0 WiFi

### DISCUSSION THREAD

**Discussion by gerthalberg** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1641] [Q&A]
I have a TFT35 E3 V3.0 and I want to add an ESP3D connection to it. Nothing fancy just to enable me to remotely start preheat and monitor temp and a webcam during print. 
I have an ESP32 Devkit, where I have loaded the ESP3D. I have connected the ESP32 to the TFT and I'm able to start heating both bed and hotend, I can also move the head and home - however I do not receive anything the other way i.e. I do not see the temps or the current position of the head on the ESP3D UI. What do I need to do in either the TFT or ESP firmware to make it work properly?
I will add, that both the touch mode and the Marlin mode works as they should. The baud rate is set to 250000 on both the TFT and on Marlin Marlin is btw running on an SKR 1.4 Turbo
Thank's in advance.

**Comment by dielectric-coder** [comment]
I have a similar setup, i.e. TFT35 E3 V3.0 and SKR MINI E3 V2, using the firmware compiled on 8 march 2121 from  "Copy to SD Card root directory to update" it works IF: 'serial always on' option is off AND the screen is in touch mode. Hope this helps
